js/phobos.js

```diff
@@ -23,14 +23,28 @@ function addAnchors()
         // we link to the first children
         var a = items[i].querySelector('a');
         if(!a) continue;
+
+        // copy-to-clipboard
+        var el = document.createElement("i");
+        var symbol = a.name.replace(".", "");
+        var text = "import " + document.body.id + " : " + symbol + ";";
+        el.setAttribute("data-clipboard-text", text);
+        el.setAttribute("aria-hidden", true);
+        el.className = "fa fa-clipboard decl_relement";
```

Comment:
Good point - I will try "smallprints" as known from the "Improve this page" link at the top right.